pgmpy/inference/ExactInference.py

```diff
@@ -248,134 +248,104 @@ def query(
         joint=True,
         show_progress=True,
     ):
-        """
-        Parameters
-        ----------
-        variables: list
-            list of variables for which you want to compute the probability
+    """
+    Parameters
+    ----------
+    variables: list
+        list of variables for which you want to compute the probability
```

Comment:
Could you please change this to: `List of variables for which to compute the posterior distribution`